configs/config_mbedtls.h

```diff
@@ -0,0 +1,166 @@
+//
+// mbedTLS config
+//
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_CRYPTO_CONFIG__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_CRYPTO_CONFIG__
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Minimal Suite-B configuration
+ *
+ * Inherited from https://github.com/ARMmbed/mbedtls/blob/1e14827bebc8480786498f007304bd852ef953c6/configs/config-suite-b.h.
+ * Some of the configurations are disabled due to incompatibility with what we need. The configs that are disabled are still
```

Comment:
So the only change from the "standard" config is the commented lines with notes?

Reply:
Only the ones that are commented out and have notes. Apart from that, there's another section called "mbedTLS configuration extensions for KVS", which contains extra flags for our needs.